presto-main/src/main/java/com/facebook/presto/operator/OperatorContext.java

```diff
@@ -1,3 +1,17 @@
+/*
```

Comment:
This header comment seems redundant?

Reply:
Removed. Misconfigured intelliJ to automatically insert copywrite info when commit. But didn't realized that it couldn't dedup. :)